pages/index.tsx

```diff
@@ -48,44 +51,50 @@ const Blurb = styled.div`
 `;
 
 export default function Index() {
-  const [tabIndex, setTabIndex] = useState(0);
```

Comment:
I had to pretty much re-write the logic in the menu to accommodate the dynamic options depending on the EMP selected. Also, there was some inconsistent behavior when switching between EMPs that offer different tabs. Now everything works from the `selectedMenuItem` string.